drivers/led_strip/ws2812_spi.c

```diff
@@ -231,6 +240,7 @@ static const struct led_strip_driver_api ws2812_spi_api = {
 		.zero_frame = WS2812_SPI_ZERO_FRAME(idx),		 \
 		.num_colors = WS2812_NUM_COLORS(idx),			 \
 		.color_mapping = ws2812_spi_##idx##_color_mapping,	 \
+		.length = DT_INST_PROP(idx, chain_length),               \
```

Comment:
Maybe `px_buf_size` can be removed ?

Reply:
That deals with number of colours too so is not the same as the length

Reply:
Well, `cfg->px_buf_size = cfg->num_colors * cfg->length * 8`. So maybe we can remove one of this variables to save a few bytes in memory ?

Reply:
Removed